assets/dev/js/frontend/handlers/container/grid-container.js

```diff
@@ -0,0 +1,52 @@
+export default class GridContainer extends elementorModules.frontend.handlers.Base {
+	isActive() {
+		return elementorFrontend.isEditMode();
+	}
+
+	getDefaultSettings() {
+		return {
+			selectors: {
+				emptyView: '.elementor-empty-view',
+				gridContainer: '.e-con',
+			},
+			scssProperties: {
+				gridTemplateRows: '--e-con-grid-template-rows',
+			},
+		};
+	}
+
+	getDefaultElements() {
+		const selectors = this.getSettings( 'selectors' );
+
+		return {
+			emptyView: this.findElement( selectors.emptyView )[ 0 ],
+			gridContainer: this.$element[ 0 ],
+		};
+	}
+
+	onElementChange( propertyName ) {
+		if ( 0 === propertyName.indexOf( 'grid_rows_grid' ) ) {
+			this.updateEmptyViewHeight();
+		}
+	}
+
+	updateEmptyViewHeight() {
+		const { emptyView } = this.elements;
+
+		emptyView.style?.removeProperty( 'min-height' );
+
+		if ( ! this.doesControlHasSliderValue() ) {
+			emptyView.style.minHeight = 'auto';
+		}
+	}
+
+	doesControlHasSliderValue() {
```

Comment:
@davseve What is going on here? I am not sure why we need this function? Also, if it is really needed, can we add some JSDoc?

Reply:
Added JSDoc comment.